crypto/provider_conf.c

```diff
@@ -425,6 +414,5 @@ static int provider_conf_init(CONF_IMODULE *md, const CONF *cnf)
 
 void ossl_provider_add_conf_module(void)
```

Comment:
Oh, lovely. So they added more layers to the conf mess. Just one or two layers more and it will be glorious.

Reply:
I'm worried that it will achieve sentience :)